network/containerizer/bridgepolicy_integration_test.go

```diff
@@ -76,9 +74,7 @@ func (s *bridgePolicyStateSuite) SetUpTest(c *gc.C) {
 	var err error
 	m, err := s.State.AddMachine("quantal", state.JobHostUnits)
 	c.Assert(err, jc.ErrorIsNil)
-	s.machine = &containerizer.MachineShim{Machine: m}
-
-	s.bridgePolicy = containerizer.NewBridgePolicy(cfg(c, 13, "provider"))
```

Comment:
This was moved off the suite and created for each test, because we have to do it _after_ any spaces are created so they end up in the cache.